### PR TITLE
Bridge Hub preliminary docs

### LINKED ISSUE #5770: Add Polkadot < > Kusama Bridge guide to the Wiki
Reference - https://forum.polkadot.network/t/polkadot-kusama-bridge/2971/25

### PR DESCRIPTION
**Pull request by DrW3RK**
- [x] Add a preliminary doc on Bridge Hub
- [x] Add a preliminary doc on Polkadot <> Kusama bridge 
- [x] Add Polkadot-JS guide for using on Polkadot <> Kusama bridge

### CODE REVIEW

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/polkadot-kusama-bridge/2971/37

File: docs/learn/learn-DOT-KSM-bridge.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+slug: ../learn-dot-ksm-bridge
+---
+
+Both Polkadot and Kusama blockchain networks achieve finality through GRANDPA consensus, which
+enables trustless bridging of both the networks through their respective Bridge Hubs. Polkadot
+Bridge Hub runs a [light client of Kusama network](https://polkadot.polkassembly.io/referenda/545)
+and Kusama Bridge Hub runs a
+[light client of Polkadot network](https://kusama.polkassembly.io/referenda/354), which were both
+enabled through their respective OpenGov referenda. This trustless bridge allows Polkadot Asset Hub
+accounts to hold wrapped KSM tokens and Kusama Asset Hub accounts to hold wrapped DOT tokens.
```
Comment (acatangiu) [inline docs/learn/learn-DOT-KSM-bridge.md]: This somewhat reads that the bridged DOT and KSM are only available/usable on asset hubs. Let's rephrase. Maybe something like:

```suggestion
enabled through their respective OpenGov referenda. This trustless bridge allows Polkadot Asset Hub
to bridge in wrapped KSM tokens and Kusama Asset Hub to bridge in wrapped DOT tokens, thus making
DOT available to all Kusama parachains and KSM to all Polkadot parachains.
```

File: docs/learn/learn-guides-DOT-KSM-bridge.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    },
+    "method": "limitedReserveTransferAssets",
+    "section": "polkadotXcm"
+  }
+}
+```
+
+Once this [extrinsic](https://assethub-polkadot.subscan.io/extrinsic/6028374-2) is signed and
+submitted, it is broadcast to Polkadot Asset Hub nodes. As this is a reserve asset transfer, the DOT
+is transferred to a dedicated account on Polkadot Asset Hub and
```
Comment (acatangiu) [inline docs/learn/learn-guides-DOT-KSM-bridge.md]:
```suggestion
is transferred to the destination's sovereign account on Polkadot Asset Hub and
```

File: docs/learn/learn-guides-DOT-KSM-bridge.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+[the wrapped DOT is issued](https://assethub-kusama.subscan.io/extrinsic/6758392-0?event=6758392-1)
+as a foreign asset and deposited onto the destination account on Kusama Asset Hub. The foreign asset
+balances of any account on Kusama Asset Hub can be queried on-chain through the
+`foreignAssets`pallet as shown below.
+
+![Wrapped DOT Balance](../assets/bridge-hub/KAH-DOT-Balance.png)
+
+## Transfer KSM to Polkadot Asset Hub
```
Comment (acatangiu) [inline docs/learn/learn-guides-DOT-KSM-bridge.md:106]: This section is incomplete

**Review by acatangiu — APPROVED** [review]
Looking very good so far!

File: docs/learn/learn-guides-DOT-KSM-bridge.md (hunk 3 above)
Comment (profradha) [reply docs/learn/learn-guides-DOT-KSM-bridge.md:106]: Yeah, there is a bug in BagPipes that's messing with Kusama AssetHub's assets dropdown. Waiting for their response on the forum post. Worst case, I'll use PJS extrinsic from the tutorial you posted and push this PR today.

image

**Review by serban300 — APPROVED** [review]
LGTM !

**Review by bkontur — APPROVED** [review]
lgtm! To be 100% complete, I think it just misses the scenario for withdrawing wDOT from AssetHubKusama back to AssetHubPolkadot (the same for wKSM from AHP back to AHK).

**Comment by DrW3RK** [comment]
@bkontur Absolutely! Will do this through a follow-up PR.

**Comment by bkontur** [comment]
> @bkontur Absolutely! Will do this through a follow-up PR.

ah, my bad, better wait for 1.2.0 fellows release, I think actual K/P release does not contain withdrawal pallet_xcm part

**Comment by acatangiu** [comment]
> > @bkontur Absolutely! Will do this through a follow-up PR.
> 
> ah, my bad, better wait for 1.2.0 fellows release, I think actual K/P release does not contain withdrawal pallet_xcm part

simple AH to AH transfer of only DOT and/or KSM works both ways even without 1.2.0 using `limited_reserve_transfer_assets` - you can do it with current runtimes

**Comment by acatangiu** [comment]
> > > @bkontur Absolutely! Will do this through a follow-up PR.
> > 
> > 
> > ah, my bad, better wait for 1.2.0 fellows release, I think actual K/P release does not contain withdrawal pallet_xcm part
> 
> simple AH to AH transfer of only DOT and/or KSM works both ways even without 1.2.0 using `limited_reserve_transfer_assets` - you can do it with current runtimes

Actually, Brani is right, there's a pretty old version of pallet-xcm (4.0.0) in current AH runtimes from SDK v1.3.0 - with no support for withdrawing.

We have to wait for the runtimes v1.2.0 upgrade to be enacted.

**Comment by DrW3RK** [comment]
Got it. We will merge these preliminary docs to the Wiki and update them after 1.2 runtime is live.

**Review by filippoweb3 — APPROVED** [review]

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/polkadot-kusama-bridge/2971/41

**Pull request merged**